file/writable_file_writer.cc

```diff
@@ -187,7 +187,20 @@ IOStatus WritableFileWriter::Flush() {
     }
   }
 
-  s = writable_file_->Flush(IOOptions(), nullptr);
+  {
+    IOSTATS_TIMER_GUARD(write_nanos);
+    TEST_SYNC_POINT("WritableFileWriter::Flush:BeforeAppend");
```

Comment:
Test sync points must have unique names. This is duplicate of the sync point in line 400 inside `WriteDirect`.